addons/blocks2image/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Save blocks as image",
-  "description": "Right click the editor workspace to export blocks as SVG/PNG images.",
+  "name": "Block images",
```

Comment:
Too vague, should remain as a verb. This looks like an addon that stops images from loading.